lib/gax.js

```diff
@@ -58,6 +58,10 @@
  *   pageToken.
  * @property {boolean=} isBundling - If set to false and the call is configured
  *   for bundling, bundling is not performed.
+ * @property {boolean || BackoffSettings=} longrunning - If set to false and the
```

Comment:
nit: `{(boolean|BackoffSettings)=}` ?  (see http://usejsdoc.org/tags-type.html)

Please make sure running `npm run doc` and no errors happening.